Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM debian:jessie
```

Comment:
Suggestion: Use [`python:2.7`](https://hub.docker.com/_/python/) as the base image instead, as it already includes almost everything you're `apt-get`ting.